### ISSUE TITLE
rotating panel 

### ISSUE THREAD

**Issue by BETTEREXTENEDEDITORMOD** [BEEmod/BEE2-items#2204]
you see this new panel in both portal 1 chamber 18 
<img width="300" alt="panel" src="https://user-images.githubusercontent.com/31172185/30253549-975d6c60-963c-11e7-8e20-744f584e8c5d.PNG"> and in portal 2 chapter 7 chamber 2 
<img width="532" alt="capture" src="https://user-images.githubusercontent.com/31172185/30253556-a8efcaa4-963c-11e7-8053-e94ba02a923c.PNG"> these panels rotate in diffrent derections could you make this so its a pistion platform type item?

**Comment by MrKrunk** [comment]
Don't know if this is possible, but maybe add types to the object like buttons? 
Cube: Blank (Portalability depends on start enabled/reversed setting) 
Sphere: [Object Name Nere]
Reflector: [Object Name Here]
Inputs would rotate the panel. Input Delay objects can be used to have them alternate automatically.

**Comment by BETTEREXTENEDEDITORMOD** [comment]
my problem is that you need eight check boxes for it to work there is four positions it can be in up down left right you need another set of four to set the positions for when it's on i don't know how to fix that

**Comment by BETTEREXTENEDEDITORMOD** [comment]
you could have it so button type is up down left autodrop first cube would be right and start reversed would make it turn in all directions

**Comment by ZombiePwner1** [comment]
Yes this is possible.

**Comment by BETTEREXTENEDEDITORMOD** [comment]
will this be hard to implement?

**Comment by BETTEREXTENEDEDITORMOD** [comment]
oh no i accidentally deleted one of my comments!

**Comment by BETTEREXTENEDEDITORMOD** [comment]
ill just rewrite the commet button type will be up down left and auto drop first cube will be right start reversed with make it spin in all four directions

**Comment by ZombiePwner1** [comment]
It doesn't work like that we can't use the button type property to change which direction it goes. The way this will work is so it's two seperate items.

**Comment by BETTEREXTENEDEDITORMOD** [comment]
oh so it'l be a bit like cube colorizer but with the directions that it goes in?

**Comment by TeamSpen210** [comment]
We'd use Cube Type.

**Comment by Darmagon3** [comment]
I agree with this, it allows for gel to be more of a puzzle and also acts kind of like a flip panel on both sides of a block.
